src/foam/u2/JSFObject.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2',
+  name: 'JSFObject',
+  extends: 'foam.core.Property',
+
+  documentation: `
+    Add properties to an object allowing the server to handle it.
```

Comment:
Change to:
A Property which is only meaningful in JS. Is stored in Java as unresolved JSON.